onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperator.cpp

```diff
@@ -635,6 +635,40 @@ namespace Dml
             ));
     }
 
+    void DmlOperator::ConvertNHWCToNCHW(
+        const uint32_t& dimCount, 
```

Comment:
>& [](http://example.com/codeflow?start=22&length=1)

Why pass by reference? It's not being updated.

Reply:
No needed, updating in next commit